src/Components/Web/src/Forms/InputBase.cs

```diff
@@ -226,24 +226,25 @@ public override Task SetParametersAsync(ParameterView parameters)
                     $"{nameof(Forms.EditContext)} dynamically.");
             }
 
-            SetAdditionalAttributesIfValidationFailed();
+            UpdateAdditionalValidationAttributes();
```

Comment:
The js files shouldn't be changing. Is that accidental?

Reply:
The build updated them, so I included them.
Should I remove them from the PR ?

Reply:
At some point, we deleted blazor.webassembly.js from source control but it seems to have made it's way back. If you can please remove it (either as part of this PR or separately). I guess leave the server.js file in, it's not worth fighting automation.